gnutls-v3.5.0/Readme.md

```diff
@@ -0,0 +1,44 @@
+Finds a SEGV error during parsing of a corrupted certificate, in
+[GnuTLS](https://www.gnutls.org/) v.3.5.0. Fixed in [da4c7a39](https://gitlab.com/gnutls/gnutls/commit/da4c7a393d273076af4f650f6cb1fd6329078824).
+
+POC
+===
+```
+../BUILD/bin/certtool --verify-chain --load-ca-certificate ca_chain.pem --infile full_pem_chain.pem
+```
+
+(To attempt replication with the respective libFuzzer driver, run with
+`ASAN_OPTIONS=detect_leaks=0` and a starting corpus of certificates in PEM
```

Comment:
If there is a leak that happens frequently we can't use it as a benchmark, as it will exhaust all memory quickly.